unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/WearableGridController.cs

```diff
@@ -370,12 +370,13 @@ private WearableGridItemModel ToWearableGridModel(WearableItem wearable)
                 Rarity = rarity,
                 Category = wearable.data.category,
                 ImageUrl = wearable.ComposeThumbnailUrl(),
-                IsEquipped = IsEquipped(wearable.id),
+                IsEquipped = IsEquipped(new UrnAndTokenId(wearable.id).Urn),
```

Comment:
Instead of allocating a temporary object for solving the urn, its preferible to make a static call or extension method